static/app/views/starfish/views/spanSummaryPage/sampleList/sampleInfo/index.tsx

```diff
@@ -15,32 +14,20 @@ type Props = {
 function SampleInfo(props: Props) {
   const {groupId, transactionName} = props;
 
-  const {data: spanTransactions} = useSpanTransactions({group: groupId});
-  const {data: spanMetrics} = useSpanTransactionMetrics(
-    {group: groupId},
-    spanTransactions.map(row => row.transaction)
-  );
-
-  const totalTimeSpent = spanTransactions.reduce(
-    (acc, row) => acc + spanMetrics[row.transaction]?.['sum(span.self_time)'],
-    0
-  );
-  const spansPerSecond = spanMetrics[transactionName]?.spans_per_second;
-  const p95 = spanMetrics[transactionName]?.p95;
-  const span_total_time = spanMetrics[transactionName]?.total_time;
+  const {data: spanMetrics} = useSpanMetrics({group: groupId}, {transactionName});
 
   return (
     <BlockContainer>
       <Block title={DataTitles.throughput}>
-        <ThroughputCell throughputPerSecond={spansPerSecond} />
+        <ThroughputCell throughputPerSecond={spanMetrics?.['spm()']} />
       </Block>
       <Block title={DataTitles.p95}>
-        <DurationCell milliseconds={p95} />
+        <DurationCell milliseconds={spanMetrics?.['p95(span.duration)']} />
       </Block>
       <Block title={DataTitles.timeSpent}>
         <TimeSpentCell
-          formattedTimeSpent={formatPercentage(span_total_time / totalTimeSpent)}
-          totalSpanTime={span_total_time}
+          formattedTimeSpent={formatPercentage(spanMetrics?.['time_spent_percentage()'])}
+          totalSpanTime={spanMetrics?.['sum(span.duration)']}
```

Comment:
If the view previous to the panel (span summary endpoint list table) is showing time spent percent scoped just to this span, is that the value we should continue here?

Reply:
Hm, what do you mean? The metrics here are fetched _for_ the span, does using `time_spent_percentage(local)` have any meaning in that context?

Reply:
Copypasta from Slack by tl;dr this turned out tricky. I need the call to return the percentage time taken by span in transaction with respect to other transactions, and `local` won't have the right behaviour, since it only changes the calculation denominator. When only fetching one transaction, it always returns 100%. I'm going to come back to this on Monday!